public/stylesheets/site/2.0/13-group-blurb.css

```diff
@@ -335,9 +335,12 @@ blurbs on the manage collection items pages, mostly reseting styles inherited fr
 	top: 28px;
 }
 
+.bookmark div.user, .bookmark div.recent {
```

Comment:
We don't need it on `li.user`, which is was also getting applied to without the div.